chaincode/main.go

```diff
@@ -49,7 +49,15 @@ func (t *SubstraChaincode) Init(stub shim.ChaincodeStubInterface) peer.Response
 
 // Invoke is called per transaction on the chaincode.
 func (t *SubstraChaincode) Invoke(stub shim.ChaincodeStubInterface) peer.Response {
+	resp, _, err := t._Invoke(stub)
 
+	if err != nil {
+		return formatErrorResponse(err)
+	}
+	return shim.Success(resp)
+}
+
+func (t *SubstraChaincode) _Invoke(stub shim.ChaincodeStubInterface) ([]byte, *Event, error) {
```

Comment:
[CS] Go private function should be lower case.
- CapitalCase function means package exported function
- lowerCase means package internal function
no need to prefix with _ is not idiomatic go

Reply:
Oh yeah, this is very very WIP, I didn't think too much, I was just trying to make things work.

Speaking of exporting functions, I tried to move the code related to the standalone functionality to a new package named "standalone", but this doesn't work: I can't reference anything from the "main" package inside the "standalone" package. 

I was thinking we could create a "chaincode" package and move everything currently in main (except the actual main function) there. This could possibly be done prior to and independently from this PR. 

Then :

- We should be in a better place to introduce a "standalone" package
- over time (in future PRs) we can slowly extract meaningful packages out of the "chaincode" package

Reply:
Or we can just keep stuffing stuff in main (like I did in this PR)...